cmake/TestSetup.cmake

```diff
@@ -11,11 +11,40 @@
 # the public, perform publicly and display publicly, and to permit others to do so.
 #=========================================================================================
 
-# Seaches for the python interpreter
-message(STATUS "Searching for python 3.6, required to run regression tests.")
-include(FindPython)
```

Comment:
Thanks @nmsriram for all your work, just a heads up though by removing this module `include(FindPython)` some other problems have been introduced. There are several variables that are defined in FindPython that are populated after calling find_package, by removing it you have to be careful because these variables are now no longer available. E.g. Python_Interpreter_FOUND is no longer defined even though Python3 3.6 is found.

Reply:
Shouldn't it be [`Python3_Interpreter_FOUND`](https://cmake.org/cmake/help/latest/module/FindPython3.html?highlight=findpython3#result-variables)?

Reply:
Yes good catch.